samples/editors/date-picker/date-limits/App.razor

```diff
@@ -0,0 +1,24 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="container vertical">
+    <div class="container vertical">
+        <DatePicker @ref="DatePicker" Height="50px" Width="220px" 
+                    Value="@SelectedDate"
+                    MinDate="@MinDate"
+                    MaxDate="@MaxDate"/>
+    </div>
+</div>
+
+@code {
+
+    public DateTime SelectedDate { get; set; }
+    public DateTime MinDate = new DateTime(DateTime.Today.Year, DateTime.Today.Month, 1);
+    public DateTime MaxDate = new DateTime(DateTime.Today.Year, DateTime.Today.Month, DateTime.DaysInMonth(DateTime.Today.Year, DateTime.Today.Month));
```

Comment:
This kind of initialization of variables should always be done in OnInitialized()
```
var current = DateTime.Today;
var dayLimit = DateTime.DaysInMonth(current.Year, current.Month);
this.MinDate = new DateTime(current.Year, current.Month, 1);
this.MaxDate = new DateTime(current.Year, current.Month, dayLimit );
```

Reply:
moved to OnInitialized, but I'm keeping the entire month instead of 14 days.